sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStore.java

```diff
@@ -36,20 +37,43 @@ public class JedisRedisCheckpointStore implements CheckpointStore {
      * This method returns the list of partitions that were owned successfully.
      *
      * @param requestedPartitionOwnerships List of partition ownerships from the current instance
-     * @return Null
+     * @return Flux of PartitionOwnership objects
      */
     @Override
     public Flux<PartitionOwnership> claimOwnership(List<PartitionOwnership> requestedPartitionOwnerships) {
-        return Flux.fromIterable(new ArrayList<>());
-    }
 
+        return Flux.fromIterable(requestedPartitionOwnerships).handle(((partitionOwnership, sink) -> {
+            String partitionId = partitionOwnership.getPartitionId();
+            String key = keyBuilder(prefixBuilder(partitionOwnership.getFullyQualifiedNamespace(), partitionOwnership.getEventHubName(), partitionOwnership.getConsumerGroup()), partitionId);
+            try (Jedis jedis = jedisPool.getResource()) {
```

Comment:
nit: between variable declarations into flow control like this and (between line 50 and 51) add a new line. This is hard for me to read because it looks like a giant block of prose.

Reply:
Added in new lines wherever I noticed this. I'll keep it in mind going forward too!